apps/showcase/src/components/utilities/otter-picker/otter-picker-pres.template.html

```diff
@@ -0,0 +1,12 @@
+<div ngbDropdown class="d-inline-block w-100">
+  <button type="button" class="btn btn-outline-primary otter-dropdown w-100" [class.selected]="!!selectedOtter()" [id]="id" ngbDropdownToggle>
+    <div class="w-100" *ngIf="selectedOtter()"><img [src]="selectedOtter()" width="34" height="34" /></div>
+    <span *ngIf="!selectedOtter()">Pick an icon</span>
+  </button>
+  <div ngbDropdownMenu [attr.aria-labelledby]="id">
+    <button ngbDropdownItem (click)="selectedOtter.set(''); onChanges(''); onTouched()">None</button>
```

Comment:
maybe we can wrap that `selectedOtter.set(''); onChanges(''); onTouched()` in a method as it's reuse the line below